### DISCUSSION TITLE
Terminal Markdown Read/Write Workflow?

### DISCUSSION THREAD

**Discussion by erwin** [ekickx/clipboard-image.nvim#26] [Q&A]
It seems like the pieces for reading and writing Markdown documents in the terminal while the actual image is visible in the terminal, is finally getting close.

Hopefully within a few years we'll have it working over Tmux, Mosh & SSH on remote machines, but for now it seems like it should at least be feasible to do locally.

It seems like we've got a lot of the pieces:

My favorite Terminal Kitty has it's [own Terminal Graphics Protocol](https://sw.kovidgoyal.net/kitty/graphics-protocol/).

There's [libsixel](https://github.com/saitoha/libsixel).

There's [imgcat in iTerm2](https://iterm2.com/documentation-images.html)

There's [tycat in terminology](https://www.enlightenment.org/docs/apps/terminology.md#Other_Useful_Features)

There are apps that bypass the terminal and just overlay the images like [ueberzug](https://github.com/seebye/ueberzug). 

There are some relevant Neovim issues:
* https://github.com/neovim/neovim/issues/12991
* https://github.com/neovim/neovim/issues/4500

Anyone aware of a local workflow that puts the pieces together?

**Comment by ekickx** [comment]
I don't really know about it but maybe you can check this plugin, [hologram.nvim](https://github.com/edluffy/hologram.nvim)

**Comment by Vaisakhkm2625** [comment]
[image.nvim](https://github.com/3rd/image.nvim)
